java/client/src/test/java/glide/managers/CommandManagerTest.java

```diff
@@ -284,15 +292,141 @@ public void prepare_request_with_slot_key_routes(SlotType slotType) {
     public void prepare_request_with_unknown_route_type() {
         CompletableFuture<Response> future = new CompletableFuture<>();
         when(channelHandler.write(any(), anyBoolean())).thenReturn(future);
-        var command =
-                Command.builder()
-                        .requestType(Command.RequestType.CUSTOM_COMMAND)
-                        .route(() -> false)
-                        .build();
 
         var exception =
                 assertThrows(
-                        IllegalArgumentException.class, () -> service.submitNewCommand(command, r -> null));
+                        IllegalArgumentException.class,
+                        () ->
+                                service.submitNewCommand(
+                                        CUSTOM_COMMAND, new String[0], Optional.of(() -> false), r -> null));
         assertEquals("Unknown type of route", exception.getMessage());
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @EnumSource(CommandManager.RequestType.class)
+    public void submitNewCommand_covers_all_mapRequestTypes(CommandManager.RequestType requestType) {
+
+        // setup
+        RedisRequestOuterClass.RequestType protobufRequestType;
+        switch (requestType) {
```

Comment:
If you will update enum as I mentioned, you can delete this `switch`

Reply:
Fixed.